src/app/Components/Details/Note/index.jsx

```diff
@@ -39,15 +44,37 @@ export default class Note extends Component {
   };
 
   render() {
+    const linkStyle = {
+      textDecoration: 'underline',
+      border: 'none',
+      background: 'transparent',
+      fontSize: '100%',
+      color: '#00664f'
+    };
     const { note } = this.props;
-
+    let noteComponent = '';
+    if (
+      note &&
+      note.type === 'document' &&
+      (note.system === 'UHW' || note.system === 'COMINO')
+    ) {
+      noteComponent = (
+        <a href="#/">
+          <strong style={linkStyle}>{note.title}</strong>
+        </a>
```

Comment:
let's try removing the inline styles and moving the `a` tags into the `strong` tags